vms/platformvm/txs/executor/staker_tx_verification.go

```diff
@@ -762,3 +712,26 @@ func verifyTransferSubnetOwnershipTx(
 
 	return nil
 }
+
+// Ensure the proposed validator starts after the current time
+func verifyStakerStartTime(chainTime, stakerTime time.Time) error {
+	if !chainTime.Before(stakerTime) {
+		return fmt.Errorf(
+			"%w: %s >= %s",
+			ErrTimestampNotBeforeStartTime,
+			chainTime,
+			stakerTime,
+		)
+	}
+	return nil
+}
+
+func verifyStakerStartsSoon(chainTime, stakerStartTime time.Time) error {
+	// Make sure the tx doesn't start too far in the future. This is done last
+	// to allow the verifier visitor to explicitly check for this error.
```

Comment:
I think the `This is done last to allow the verifier visitor to explicitly check for this error` comment is wrong now that it's in a helper. We should restore that comment to wherever we call this function and remove it here

Reply:
Done